inc/classes/traits/trait-updater-api.php

```diff
@@ -0,0 +1,97 @@
+<?php
+namespace WP_Rocket\Traits;
+
+use WP_Rocket\Logger\Logger;
+
+/**
+ * Trait for the plugin updater.
+ *
+ * @since  3.3.6
+ * @author Grégory Viguier
+ */
+trait Updater_Api_Subscriber {
```

Comment:
```suggestion
trait Updater_Api_Tools {
```

Reply:
The file should be renamed into `trait-updater-api-tools` then, shouldn't it?

Reply:
It would be best yes